code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -118,13 +121,6 @@
 		return
 	return ..()
 
-/mob/living/simple_animal/bot/mulebot/Cross(atom/movable/AM)
-	. = ..()
-	if(ishuman(AM))
-		RunOver(AM)
```

Comment:
Hehe... Funnily enough I tested this same thing when I rewrote most of this code.